RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -175,6 +220,17 @@
     src           = 'jetCoreRegionalStepTrackCandidates',
     Fitter        = 'FlexibleKFFittingSmoother'
 )
+jetCoreRegionalStepBarrelTracks = jetCoreRegionalStepTracks.clone(
+    AlgorithmName = 'jetCoreRegionalStep',
+    src           = 'jetCoreRegionalStepBarrelTrackCandidates',
+    Fitter        = 'FlexibleKFFittingSmoother'
```

Comment:
```suggestion
    src           = 'jetCoreRegionalStepBarrelTrackCandidates',
```
please set a parameter only where change is needed